docs/topics/proc_using-a-nexus-repository-server-on-a-openshiftlocal.adoc

```diff
@@ -1,137 +1,157 @@
+
+[discrete]
+== Procedure
 
 . Log in to your {OpenShiftLocal} instance.
 +
-[source,bash,subs="attributes+"]
 --
+[source,bash,subs="attributes+"]
+----
 oc login https://{osl-login-url} -u developer -p developer
---
-+
-. You can reuse the Docker daemon instance used by {OpenShiftLocal} to download the latest versions of the Nexus Docker container image.
-+
+----
+
+You can reuse the Docker daemon instance used by {OpenShiftLocal} to download the latest versions of the Nexus Docker container image.
+
 [source,bash,subs="attributes+"]
---
+----
 eval $(minishift docker-env)
 docker pull openshift/jenkins-2-centos7
 docker pull openshiftio/launchpad-jenkins-slave
 docker pull sonatype/nexus
+----
 --
 
-== Setting up the Nexus Application
-
-.  Create a new project to contain the Nexus server. You can also use the _New Project_ button on the Web console to do this.
+.  Create a new project to contain the Nexus server.
+You can also use the _New Project_ button on the Web console to do this.
 +
 [source,bash,subs="attributes+"]
---
+----
 oc new-project {nexus-project-name}
---
-+
+----
+
 . Deploy the Nexus container image.
 +
 [source,bash,subs="attributes+"]
---
+----
 oc new-app sonatype/nexus
---
-+
+----
+
 . Expose the service route URL of the Nexus server.
 +
 [source,bash,subs="attributes+"]
---
+----
 oc expose svc/nexus
---
-+
+----
+
 . Attach a persistent volume claim with a minimum size of 10 GiB to the pod running your Nexus application.
 +
+--
 // NOTE: seems that minishift (with oc v 3.6.0alpha1) is fine with USING '1G'  for volume size, while openshift Online requires the syntax '1Gi'.
-+
+
 [source,bash,subs="attributes+"]
---
+----
 oc volumes dc/nexus --add --name 'nexus-volume-1' --type 'pvc' --mount-path '/sonatype-work/' --claim-name 'nexus-pv' --claim-size '10Gi' --overwrite
+----
 --
-+
+
 .  Navigate to the project containing your booster application.
 +
 [source,bash,subs="attributes+"]
---
+----
 oc project {project-name}
---
-+
-. Define and start a new build. Pass in a parameter to set the value of the `MAVEN_MIRROR_URL` to match the service URL of your Nexus application:
+----
+
+. Define and start a new build.
 +
-[NOTE]
 --
+Pass in a parameter to set the value of the `MAVEN_MIRROR_URL` to match the service URL of your Nexus application:
+
+[NOTE]
+====
 Ensure that the YAML template of the builder image you are using for your application has the `MAVEN_MIRROR_URL` environmental variable defined.
 If it does not, see the Nexus documentation for instructions on link:https://help.sonatype.com/display/NXRM2/Maven+and+Other+Build+Tools[configuring your build manually^] before proceeding.
---
-+
+====
+
 [source,bash,subs="attributes+"]
---
+----
 $ oc new-build {app-name}:latest~{scm-repo-url} \
 -e MAVEN_MIRROR_URL='http://nexus.{nexus-project-name}:8081/nexus/content/groups/public'
+----
+
+Nexus comes pre-configured for the Maven Central Repository, but you may need other repositories for your application.
```

Comment:
If we're changing casing, might as well lower-case "Repository", which isn't part of the name. "Maven Central" is a name. This should IMHO read "Maven Central repository".

Reply:
Fixed.